test/sideinput-e2e/sideinput-e2e_test.go

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
We exclude the e2e test from go build tool.
```suggestion
//go:build test

/*
```